keras_nlp/src/version_utils.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2023 The KerasNLP Authors
+# Copyright 2021 The KerasNLP Authors
```

Comment:
Do we want to go back in time? :D

Reply:
oops yeah was just trying to make the diff here go away, but let me update everything